percona-server-5.7/Dockerfile-dockerhub

```
@@ -0,0 +1,81 @@
+# This Dockerfile should be used for docker official repo
+#it is intentionally used another UID, to have backward compatibility with the previous image versions published on Docker Hub
+
+FROM centos:8
+
+LABEL org.opencontainers.image.authors="info@percona.com"
+
+RUN groupdel input && userdel systemd-coredump && groupadd -g 999 mysql
+RUN useradd -u 999 -r -g 999 -s /sbin/nologin \
+        -c "Default Application User" mysql
```

can we keep one style?
```suggestion
RUN groupdel input; \
    userdel systemd-coredump; \
    groupadd -g 999 mysql; \
    useradd -u 999 -r -g 999 -s /sbin/nologin \
        -c "Default Application User" mysql
```